packages/smooth_app/lib/views/bottom_sheet_views/faq_handle_view.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_list_tile.dart';
+import 'package:smooth_app/helpers/launch_url_helper.dart';
+
+class FaqHandleView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: SizedBox(
+        height: MediaQuery.of(context).size.height * 0.8,
+        child: ListView(
+          shrinkWrap: true,
+          scrollDirection: Axis.vertical,
+          children: <Widget>[
+            buildContainer(context),
+            SmoothListTile(
+              text: AppLocalizations.of(context)!.faq,
+              leadingWidget: const Icon(Icons.open_in_new),
+              onPressed: () =>
+                  _launchUrl(context, 'https://world.openfoodfacts.org/faq'),
+            ),
+            SmoothListTile(
+              text: AppLocalizations.of(context)!.discover,
+              leadingWidget: const Icon(Icons.open_in_new),
+              onPressed: () => _launchUrl(
+                  context, 'https://world.openfoodfacts.org/discover'),
+            ),
+            SmoothListTile(
+              text: AppLocalizations.of(context)!.how_to_contribute,
+              leadingWidget: const Icon(Icons.open_in_new),
+              onPressed: () => _launchUrl(
+                  context, 'https://world.openfoodfacts.org/contribute'),
+            ),
+            SmoothListTile(
+              text: AppLocalizations.of(context)!.support,
+              leadingWidget: const Icon(Icons.open_in_new),
+              onPressed: () =>
+                  _launchUrl(context, 'https://support.openfoodfacts.org/help'),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget buildContainer(BuildContext context) {
```

Comment:
buildHeader or something would probably fit better. Also you could build a own global `ModalBottomSheetHeader` Widget for all bottom sheets

Reply:
Changes done. Thanks for the suggestions.